src/wp-includes/view-transitions.js

```diff
@@ -0,0 +1,191 @@
+// This script should eventually live elsewhere, but for now it's just in `wp-includes` for simplicity.
```

Comment:
probably goes in src/js/_enqueues/admin/ (plus add it to `Gruntfile.js` in the admin-js/files section)

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/8370/commits/816315910a9147899a5cff75cdcdb092dadffeb8